pages/documentation/simulation/timedomain/dynawo/dynamic-models-dsl.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+---
+
+# Dynawo - Dynamic Models DSL
+
+The Dynamic Models DSL is a domain specific language written in groovy for the association of dynamic models to each static equipment present in the network. If some equipment are not configured, Dynawo would use a default model and set of parameters.
+
+* TOC
+{:toc}
+
+## BlackBoxModel
+Most of the models supported are `BlackBoxModels`. This kind of dynamic models have three attributes:
+- `staticId` refers to the ID of the load in the network
+- `dynamicModelId` is an optional attribute to identify this model. By default, it is equal to the `staticId`.
+- `parameterSetId` refers a set of parameters for this model in one of the network parameters file
+
+## Generators
+All the following models are [BlackBoxModels](#blackboxmodel) you can associate to an IIDM generator. To do so, you have to use their name as a keyword and define at least a `staticId` and a `parameterSetId`. Like every other `BlackBoxModel`, the `dynamicModelId` is optional and would be equal to the `staticId` if not set.
+
+Supported models are:
+- GeneratorSynchronousThreeWindings
+- GeneratorSynchronousThreeWindingsProportionalRegulations
+- GeneratorSynchronousFourWindings
+- GeneratorSynchronousFourWindingsProportionalRegulations
+
+**Example**
+```groovy
+GeneratorSynchronousThreeWindings {
+    staticId '<LOAD_ID>'
+    dynamicModelId '<DYNAMIC_MODEL_ID>'
+    parameterSetId '<PARAMETER_ID>'
+}
+GeneratorSynchronousThreeWindingsProportionalRegulations {
+    staticId '<LOAD_ID>'
+    dynamicModelId '<DYNAMIC_MODEL_ID>'
+    parameterSetId '<PARAMETER_ID>'
+}
+GeneratorSynchronousFourWindings {
+    staticId '<LOAD_ID>'
+    dynamicModelId '<DYNAMIC_MODEL_ID>'
+    parameterSetId '<PARAMETER_ID>'
+}
+GeneratorSynchronousFourWindingsProportionalRegulations {
+    staticId '<LOAD_ID>'
+    dynamicModelId '<DYNAMIC_MODEL_ID>'
+    parameterSetId '<PARAMETER_ID>'
+}
+```
+
+### GeneratorSynchronousThreeWindingsProportionalRegulations
+- GeneratorSynchronousThreeWindings
+- GeneratorSynchronousThreeWindingsProportionalRegulations
+- GeneratorSynchronousFourWindings
+- GeneratorSynchronousFourWindingsProportionalRegulations
+
```

Comment:
Is it possible that this small section is not necessary?

Reply:
I think we can remove the block, they are already referenced a few lines up

Reply:
You right, I forgot to remove it